### PR TITLE
Refactor ReportPageFooter with less HTML

### PR DESCRIPTION
**Pull request by jessabean**
### Description
Sneaking in a small refactor to reduce unnecessary nested `<div>` tags and delete unused code in the footer component.

---
### How to test
**Cloudfront link:** https://d3enky33yiekmt.cloudfront.net/
User: `stateuser@test.com`

1. Log in as a state user and navigate through **both** a WP and a SAR and verify that the "Previous" and "Continue" buttons work and look as expected. 
2. Do the same as an admin user.

_Note:_ You may see some variation between spacing above the footer compared to what is currently in production. I did this on purpose to get closer to what is defined in Figma, but need to refactor some additional pages in the future to remove padding/margin in some areas where it doesn't need to exist.

### Notes

---
### Pre-review checklist

- [x] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
- [x] I have performed a self-review of my code
- [ ] I have manually tested this PR in the deployed cloud environment

---
#### Review
- [ ] Design: This work has been reviewed and approved by design, if necessary
- [ ] Product: This work has been reviewed and approved by product owner, if necessary

#### Security
_If either of the following are true, notify the team's ISSO (Information System Security Officer)._

- [ ] These changes are significant enough to require an update to the SIA.
- [ ] These changes are significant enough to require a penetration test.
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/app-api/forms/sar.json [hunk 1]
```diff
@@ -18,8 +18,12 @@
         },
         "praDisclosure": [
           {
-            "type": "span",
-            "content": "<b>PRA Disclosure Statement</b><br/>Under the Privacy Act of 1974 any personally identifying information obtained will be kept private to the extent of the law. According to the Paperwork Reduction Act of 1995, no persons are required to respond to a collection of information unless it displays a valid OMB control number. The valid OMB control number for this information collection is 0938-1053. The time required to complete this information collection is estimated to average 2.5 hours per response, including the time to review instructions, search existing data resources, gather the data needed, and complete and review the information collection. If you have comments concerning the accuracy of the time estimate(s) or suggestions for improving this form, please write to: CMS, 7500 Security Boulevard, Attn: PRA Reports Clearance Officer, Mail Stop C4-26-05, Baltimore, Maryland 21244-1850"
+            "type": "p",
```
Comment (bangbay-bluetiger) [inline services/app-api/forms/sar.json:21]: Semantically, this should probably be a heading, but that might be a can of worms.

File: services/ui-src/src/components/reports/ReportPageFooter.tsx [hunk 2]
```diff
@@ -34,60 +29,53 @@ export const ReportPageFooter = ({
...
-              }
-            >
-              {submitting ? <Spinner size="sm" /> : "Continue"}
-            </Button>
-          )}
-        </Flex>
-      </Box>
+    <>
+      <Flex sx={sx.footerBox}>
+        {!hidePrevious ? prevButton : null}
```
Comment (bangbay-bluetiger) [inline services/ui-src/src/components/reports/ReportPageFooter.tsx:72]: Will `{!hidePrevious && prevButton}` work?

**Review by bangbay-bluetiger — APPROVED** [review]
Looks good to me.

**Review by benmartin-coforma — APPROVED** [review]
Pretty 🤩

File: services/app-api/forms/sar.json (hunk 1 above)
Comment (jessabean) [reply services/app-api/forms/sar.json:21]: Agreed...I'm hoping maybe we can figure it out as part of our semantic structure spike?

**Pull request merged**